### DISCUSSION TITLE
maven-home POM red - <maven.home>${maven.home}</maven.home>

### DISCUSSION THREAD

**Discussion by Antonioafj** [quarkusio/quarkus#41374] [Q&A]
Windows PowerShell
Copyright (C) Microsoft Corporation. Todos os direitos reservados.

Experimente a nova plataforma cruzada PowerShell https://aka.ms/pscore6

PS D:\Estudo Programacao\CODIGOS-ALEATORIOS\appquarkus> quarkus dev
[INFO] Scanning for projects...
[INFO] 
[INFO] ----------------------< tech.buildrun:appquarkus >----------------------
[INFO] Building appquarkus 1.0.0-SNAPSHOT
[INFO]   from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- quarkus:3.10.2:dev (default-cli) @ appquarkus ---
[INFO] Invoking resources:3.3.1:resources (default-resources) @ appquarkus
[INFO] Copying 2 resources from src\main\resources to target\classes
[INFO] Invoking quarkus:3.10.2:generate-code (default) @ appquarkus
[INFO] Invoking compiler:3.12.1:compile (default-compile) @ appquarkus
[INFO] Recompiling the module because of changed source code.
[INFO] Compiling 1 source file with javac [debug release 21] to target\classes
[INFO] Annotation processing is enabled because one or more processors were found
  on the class path. A future release of javac may disable annotation processing
  unless at least one processor is specified by name (-processor), or a search
  path is specified (--processor-path, --processor-module-path), or annotation
  processing is enabled explicitly (-proc:only, -proc:full).
  Use -Xlint:-options to suppress this message.
  Use -proc:none to disable annotation processing.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /D:/Estudo Programacao/CODIGOS-ALEATORIOS/appquarkus/src/main/java/tech/buildrun/controller/UserController.java:[17,34] ')' or ',' expected
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  8.039 s
[INFO] Finished at: 2024-06-23T12:50:57-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.10.2:dev (default-cli) on project appquarkus: Unable to execute mojo: Compilation failure
[ERROR] /D:/Estudo Programacao/CODIGOS-ALEATORIOS/appquarkus/src/main/java/tech/buildrun/controller/UserController.java:[17,34] ')' or ',' expected
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
PS D:\Estudo Programacao\CODIGOS-ALEATORIOS\appquarkus> 
![maintask gradlle](https://github.com/quarkusio/quarkus/assets/167789057/19d92042-5572-4acf-b55d-c7e34def4d8f)
![mavenhome](https://github.com/quarkusio/quarkus/assets/167789057/9264438b-5bfe-4922-b333-764a8e34eae0)
[pomquarkus.txt](https://github.com/user-attachments/files/15945404/pomquarkus.txt)

[a comment by an automated account is not shown]

**Comment by gastaldi** [comment]
Your `UserController` class is missing a `)` or a `;`. Please check the error message

[a comment by an automated account is not shown]
